src/components/Dashboard/PendingTxListItem/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement } from 'react'
+import { Text } from '@gnosis.pm/safe-react-components'
+import { TransactionSummary } from '@gnosis.pm/safe-react-gateway-sdk'
+import { Link } from 'react-router-dom'
+import ChevronRightIcon from '@material-ui/icons/ChevronRight'
+import styled from 'styled-components'
+
+import { useAssetInfo } from 'src/routes/safe/components/Transactions/TxList/hooks/useAssetInfo'
+import { useKnownAddress } from 'src/routes/safe/components/Transactions/TxList/hooks/useKnownAddress'
+import { useTransactionType } from 'src/routes/safe/components/Transactions/TxList/hooks/useTransactionType'
+import { getTxTo } from 'src/routes/safe/components/Transactions/TxList/utils'
+import { boldFont, grey400, primary200, sm } from 'src/theme/variables'
+import { isMultisigExecutionInfo } from 'src/logic/safe/store/models/types/gateway.d'
+import Spacer from 'src/components/Spacer'
+import { CustomIconText } from 'src/components/CustomIconText'
+import { TxInfo } from 'src/routes/safe/components/Transactions/TxList/TxCollapsed'
+import { EMPTY_DATA } from 'src/logic/wallets/ethTransactions'
+
+const TransactionToConfirm = styled(Link)`
+  min-width: 270px;
+  height: 40px;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 25px auto 2fr 1.5fr;
+  gap: 4px;
+  margin: ${sm} auto;
+  padding: 4px 16px;
+  text-decoration: none;
+  background-color: ${({ theme }) => theme.colors.white};
+  border: 2px solid ${grey400};
+  color: ${({ theme }) => theme.colors.text};
+  border-radius: 8px;
+`
+
+const StyledConfirmationsCount = styled.div`
+  padding: 8px 6px;
```

Comment:
I would suggest slightly more horizontal padding i.e. `8px 12px` and a font-size of 12px